core/src/main/java/me/lokka30/treasury/plugin/core/hooks/placeholder/PlaceholdersExpansion.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * This file is/was part of Treasury. To read more information about Treasury such as its licensing, see <https://github.com/ArcanePlugins/Treasury>.
+ */
+
+package me.lokka30.treasury.plugin.core.hooks.placeholder;
+
+import java.util.List;
+import me.lokka30.treasury.plugin.core.hooks.PlayerData;
+
+public abstract class PlaceholdersExpansion {
+
+    /**
+     * Request player data for balance caching.
+     *
+     * @return player data
+     */
+    public abstract List<PlayerData> requestPlayerData();
```

Comment:
This method name and javadoc sounds like it's going to return a singular player data object but it returns a list. We should change it to something that aligns better with the return type.

Reply:
And that's why I don't do draft reviews, neither I want them without me asking for such. You haven't seen any implementation of this, judging by name is gonna mislead you a lot.

Reply:
I don't see why reviews should wait until every ounce of work is done. Rather, thoughts can be shared during the process of working on an objective and it may reduce the amount of wasted work if someone has something useful to say.